jwql/website/apps/jwql/views.py

```diff
@@ -649,15 +679,21 @@ def query_submit(request):
     filters = anomaly_query_config.FILTERS_CHOSEN
     exposure_time_min = anomaly_query_config.EXPTIME_MIN
     exposure_time_max = anomaly_query_config.EXPTIME_MAX
-    observing_modes = anomaly_query_config.OBSERVING_MODES_CHOSEN
+    detectors = anomaly_query_config.DETECTORS_CHOSEN
     anomalies = anomaly_query_config.ANOMALIES_CHOSEN_FROM_CURRENT_ANOMALIES
+    readpatts = anomaly_query_config.READPATTS_CHOSEN
+    exptypes = anomaly_query_config.EXPTYPES_CHOSEN
+    gratings = anomaly_query_config.GRATINGS_CHOSEN
     parameters = {}
     parameters['instruments'] = instruments
     parameters['apertures'] = apertures
     parameters['filters'] = filters
-    parameters['observing_modes'] = observing_modes
+    parameters['detectors'] = detectors
     parameters['exposure_time_min'] = exposure_time_min
     parameters['exposure_time_max'] = exposure_time_max
+    parameters['exposure_types'] = exptypes
+    parameters['read_patterns'] = readpatts
+    parameters['gratings'] = gratings
     parameters['anomalies'] = anomalies
```

Comment:
This comes across as not very D.R.Y. to me, would it be ok to just reference `anomaly_query_config` directly instead of creating separate variables?  Something like:

```python
parameters = {}
parameters['instrument'] = anomaly_query_config.INSTRUMENTS_CHOSEN
parameters['apertures'] = anomaly_query_config.APERTURES_CHOSEN
... etc.
```

And then whenever you need to access, say the filters, you could just use `parameters['filters']` instead?

Reply:
Yes, that would be much cleaner!